dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/MaxOpenConnectionsHandler.java

```diff
@@ -49,7 +46,7 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
             // notify every 10 minutes if max concurrent conns was hit
             long currentTime = System.currentTimeMillis();
             if (currentTime > (lastConnExceededTime + 600000L)) {
-                String channelName = ctx.channel().attr(ConfigConstants.NameKey).get();
+                String channelName = ctx.channel().attr(ConfigConstants.NAME_KEY).get();
```

Comment:
The Netty Channel already has a overriden toString which concats the host port of local and remote but we use these names to mimic the CHFW behavior related to logging. We use attributes to append to a Channel in this case instead of an extension to the class mainly cause it's a cleaner/easier solution IMO

Reply:
OK